### PR TITLE
Python 3.6.4 + 3.7.0: add extension mpmath

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-3.7.0-intel-2018b.eb [hunk 1]
```diff
@@ -218,6 +218,9 @@ exts_list = [
         'source_urls': ['https://pypi.python.org/packages/source/p/py_expression_eval'],
         'checksums': ['b9662c58f8835f6fa3380990f870429fe1176008718a47ce054a7867c4091ad8'],
     }),
+    ('mpmath', '1.0.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/mpmath'],
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.7.0-intel-2018b.eb]: @smoors checksum is missing here?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3123.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/5c3509fdec2a1907504604281500e17c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2010.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2861629eb3b3c44f5483adb7fd59d227 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 30 out of 30 (8 easyconfigs in this PR)
node2603.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1961a2fd6bb8ac9f812f82cd7b42348d for a full test report.

**Comment by akesandgren** [comment]
And take note of https://github.com/easybuilders/easybuild-easyconfigs/issues/6682

**Comment by boegel** [comment]
@akesandgren That's pretty much irrelevant to this PR, since I don't plan to remove the Python 3.7.0 easyconfigs (see https://github.com/easybuilders/easybuild-easyconfigs/issues/6682#issuecomment-413003200)

**Comment by boegel** [comment]
@smoors Necessary changes made in https://github.com/smoors/easybuild-easyconfigs/pull/4, please merge that PR to update this PR?

**Pull request merged**